integration_test/third_party_apps_data/applications/mongodb/debian_ubuntu/post

```diff
@@ -0,0 +1,3 @@
+set -e
+
+sudo systemctl restart mongod
```

Comment:
Does mongod need to be restarted here?

Reply:
Mongo by default install does not start I believe.

Spiked out a fresh run on a new VM and this line at least starts the dead/inactive service.
```
Created symlink /etc/systemd/system/multi-user.target.wants/mongod.service → /lib/systemd/system/mongod.service.
keith_schmitt_bluemedora_com@keith-testing-mongo-debian-11:~$ mongosh
Current Mongosh Log ID:	61fbef1c2f64cf0750e81915
Connecting to:		mongodb://127.0.0.1:27017/?directConnection=true&serverSelectionTimeoutMS=2000&appName=mongosh+1.1.9
MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017
keith_schmitt_bluemedora_com@keith-testing-mongo-debian-11:~$ mongo
MongoDB shell version v5.0.5
connecting to: mongodb://127.0.0.1:27017/?compressors=disabled&gssapiServiceName=mongodb
Error: couldn't connect to server 127.0.0.1:27017, connection attempt failed: SocketException: Error connecting to 127.0.0.1:27017 :: caused by :: Connection refused :
connect@src/mongo/shell/mongo.js:372:17
@(connect):2:6
exception: connect failed
exiting with code 1
keith_schmitt_bluemedora_com@keith-testing-mongo-debian-11:~$ sudo systemctl status mongod
● mongod.service - MongoDB Database Server
     Loaded: loaded (/lib/systemd/system/mongod.service; enabled; vendor preset: enabled)
     Active: inactive (dead)
       Docs: https://docs.mongodb.org/manual
```

Reply:
Huh, okay. That's weird on the part of the mongodb package - Debian policy says that daemons should be enabled and started automatically by package installation unless they require user configuration before they can run (and you don't appear to have that here). I guess that's the danger of using non-distribution packages. You could simplify the install script to `systemctl enable --now mongod` to combine enable and start.